climpred_skeleton/alignment.py

```diff
@@ -0,0 +1,21 @@
+import xarray as xr
+
+from .comparison import Comparison
+
+
+class LeadAlignment(Comparison):
+    def __init__(
+        self,
+        prediction: xr.Dataset,
+        observation: xr.Dataset,
+        compare: str,
+        alignment: str,
+    ):
+        super().__init__(prediction, observation)
+        self.alignment = alignment
+        self.compare = compare
+        obj = super().factory(self.compare)
+        self.prediction, self.observation = obj.broadcast()
+
+    def align(self):
+        pass
```

Comment:
I'll fill this out in the next PR once we have some comments/thoughts on the current implementation.